examples/island_cluster_with_vm_router/vars.tf

```diff
@@ -0,0 +1,48 @@
+variable "project_id" {
+  type = string
+}
+
+variable "region" {
+  type = string
+}
+
+variable "node_locations" {
```

Comment:
We generally recommend hard coding this in examples as much as possible rather than providing via vars but feel free to keep it if it's needed for the blog post.

Reply:
This is required for this example, since this is used to create router VM.